content/en/blog/_posts/2022-12-08-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Retroactive Default StorageClass"
+date: 2022-12-08
+slug: retroactive-default-storage-class
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+The v1.25 release of Kubernetes introduced an alpha feature to change how a default StorageClass was assigned to a PersistentVolumeClaim (PVC).
+With the feature enabled, you no longer need to create a default StorageClass first and PVC second to assign the class. Additionally, any PVCs without a StorageClass assigned can be updated later.
+This feature was graduated to beta in Kubernetes 1.26.
+
+See [Official Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retroactive-default-storageclass-assignment) for more details.
+
+## Why did StorageClass assignment need improvements
+
+Users might already be familiar with a similar feature that assigns default StorageClasses to **new** PVCs at the time of creation. This is currently handled by the [admission controller](/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass).
+
+But what if there wasn't a default StorageClass defined at the time of PVC creation? 
+Users would end up with a PVC that would never be assigned a class. 
+As a result, no storage would be provisioned, and the PVC would be somewhat "stuck" at this point. 
+Generally, two main scenarios could result in "stuck" PVCs and cause problems later down the road. 
+Let's take a closer look at each of them.
+
+### Changing default StorageClass did not have a good solution
```

Comment:
Since we're talking about the two scenarios that could cause the *stuck* PVC situation, amending our subheading to reflect the below would be more appropriate.
```suggestion
### Changing default StorageClass
```